testing/test-test_logger/.gitignore

```diff
@@ -0,0 +1 @@
+target/
```

Comment:
Cargo won't create a `target` directory here so I think we can omit this .gitignore